lib/Fmt_ast.ml

```diff
@@ -1520,18 +1520,14 @@ and fmt_infix_op_args c ~parens xexp op_args =
            $ fmt_if_k (not last) (break 1 0) ) )
     $ fmt_if_k (not last_grp) (break 1 0)
   in
-  let opn, hint, cls =
-    if parens || Poly.(c.conf.infix_precedence = `Parens) then
```

Comment:
The `infix-precedence` option is not taken into consideration anymore, but the diff produced by test_branch.sh still looks good, only a few parens have been added.
We could remove this option since it is not used anymore anywhere else in ocamlformat.

Reply:
This option was used only here, it looks like you just un-implemented that option.